### PR TITLE
Avseenkov V HomeWork7 Ex1 LookupArray

### PR DESCRIPTION
**Pull request by a-vov4eg**
I learned how to write correct title. I full of shame for my last 6 titles. Seriosly.

### CODE REVIEW

File: homework/Avseenkov V/HomeWork7/Ex1_LookUpArrayService/Service/LookupArrayService.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            if (min >= value) {
+                min = value;
+            }
+        }
+        return min;
+    }
+
+    public int indexOfMax (int[] array) {
+        if (array.length == 0) {
+            return 0;
```
Comment (TLittleProgrammer) [inline homework/Avseenkov V/HomeWork7/Ex1_LookUpArrayService/Service/LookupArrayService.java]: Должно возвращать -1

File: homework/Avseenkov V/HomeWork7/Ex1_LookUpArrayService/Service/LookupArrayService.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                max = array[i];
+                indexOfMax = i;
+            }
+        }
+        return indexOfMax;
+    }
+
+    public int indexOfMin (int[] array) {
+        if (array.length == 0) {
+            return 0;
```
Comment (TLittleProgrammer) [inline homework/Avseenkov V/HomeWork7/Ex1_LookUpArrayService/Service/LookupArrayService.java]: Должно возвращать -1

**Review by TLittleProgrammer — CHANGES_REQUESTED** [review]
lgfm, but you hane eny little mistakes

File: homework/Avseenkov V/HomeWork7/Ex1_LookUpArrayService/Service/LookupArrayService.java (hunk 1 above)
Comment (a-vov4eg) [reply homework/Avseenkov V/HomeWork7/Ex1_LookUpArrayService/Service/LookupArrayService.java]: fixed

File: homework/Avseenkov V/HomeWork7/Ex1_LookUpArrayService/Service/LookupArrayService.java (hunk 2 above)
Comment (a-vov4eg) [reply homework/Avseenkov V/HomeWork7/Ex1_LookUpArrayService/Service/LookupArrayService.java]: fixed

File: homework/Avseenkov V/HomeWork7/Ex1_LookUpArrayService/Service/LookupArrayDemo.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package Ex1_LookUpArrayService.Service;
+
+public class LookupArrayDemo {
+    public static void main(String[] args) {
+        LookupArrayService lookupArrayService = new LookupArrayService();
+        int[] array0 = {1, 2, 3, 4, 5, 4, 3, 2, 0};
```
Comment (javaGuruBY) [inline homework/Avseenkov V/HomeWork7/Ex1_LookUpArrayService/Service/LookupArrayDemo.java:6]: give meaningful name to arrays

File: homework/Avseenkov V/HomeWork7/Ex1_LookUpArrayService/Test/LookupArrayTest.java [hunk 4]
```diff
@@ -0,0 +1,6 @@
+package Ex1_LookUpArrayService.Test;
+
+public class LookupArrayTest {
```
Comment (javaGuruBY) [inline homework/Avseenkov V/HomeWork7/Ex1_LookUpArrayService/Test/LookupArrayTest.java:3]: test?

**Review by javaGuruBY — APPROVED** [review]
left some comments, otherwise lgfm

**Review by akupreychik — APPROVED** [review]
lgfm

**Review by vladis1350 — APPROVED** [review]
Lgfm

**Review by TLittleProgrammer — APPROVED** [review]
lgfm

**Pull request merged**